docs/onboarding/llm-analytics/anthropic.tsx

```diff
@@ -63,7 +63,7 @@ export const AnthropicInstallation = (): JSX.Element => {
                         These SDKs **do not** proxy your calls. They only fire off an async call to PostHog in the
                         background to send the data. You can also use LLM analytics with other SDKs or our API, but you
                         will need to capture the data in the right format. See the schema in the [manual capture
-                        section](/docs/llm-analytics/installation/manual-capture) for more details.
+                        section](https://posthog.com/docs/llm-analytics/installation/manual-capture) for more details.
```

Comment:
Is this going to break links in posthog.com where we'll not open in a new tab? We gotta check that

Reply:
Magic:

<img width="841" height="480" alt="Screenshot 2025-12-03 at 1 16 52 PM" src="https://github.com/user-attachments/assets/a3ec6623-d730-490e-9dcf-c607ffb37f70" />